src/modal/modal.spec.ts

```diff
@@ -36,7 +25,8 @@ describe('ngb-modal', () => {
       toHaveModal: function(util, customEqualityTests) {
         return {
           compare: function(actual, content?, selector?) {
-            const allModalsContent = document.querySelector(selector || 'body').querySelectorAll('.modal-content');
+            const allModalsContent =
+                document.querySelector(selector || 'body').querySelectorAll('.modal-dialog > .modal-content');
```

Comment:
The selector here needed to be a bit strengthen as we might end-up with my changes with several nodes with the same `.modal-content` class.